print_bug_page.php

```diff
@@ -529,7 +529,9 @@
 		$c_filesize = number_format( $t_attachment['size'] );
 		$c_date_added = date( config_get( 'normal_date_format' ), $t_attachment['date_added'] );
 		if( isset( $t_attachment['icon'] ) ) {
-			echo '<img src="', $t_attachment['icon']['url'], '" alt="', $t_attachment['icon']['alt'], '" />&#160;';
+			echo '<i class="ace-icon fa fa-lg ', $t_attachment['icon']['url'], '" alt="', $t_attachment['icon']['alt'], '"></i>&#160;';
```

Comment:
Does it make sense to replace ['icon']['url'] with something more meaningful? e.g. ['icon']['name']?